scalableminds/webknossos
# Add Render Animation Worker Job

This PR adds the frontend and backend feature for starting a animation job of the current annotation / dataset.

<img width="643" alt="image" src="https://github.com/scalableminds/webknossos/assets/1105056/613637d8-0993-4dc6-a31b-0f4b109f6df3">


### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Start a worker with animation job (https://github.com/scalableminds/webknossos/issues/7338)
- Test in View mode and for an annotation
- For "Menu" in navbar select "Create Animation"
- Play around with available options to create various animations
- Restrict your orga to "Basic" plan and check plan enforcement

### Issues:
- fixes #7338

------
